distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -1142,15 +1144,15 @@ round_min_long = -99999999999999
 
 ## PRIVATE
    Restrict rounding decimal_places parameter.
-check_decimal_places : Integer -> Function
+check_decimal_places : Integer -> Function -> Any ! Illegal_Argument
```

Comment:
```suggestion
check_decimal_places : Integer -> Any -> Any ! Illegal_Argument
```

Because action is Any and is returned unless there is an error.